config/initializers/notifications.rb

```diff
@@ -0,0 +1,5 @@
+TahiNotifier.subscribe("paper::revised") do |subscription_name, payload|
```

Comment:
Unused block argument - `subscription_name`. If it's necessary, use `_` or `_subscription_name` as an argument name to indicate that it won't be used.